feature_tests/features/producer/producer-createDocumentPointer-success.feature

```diff
@@ -160,3 +161,73 @@ Feature: Producer Create Success scenarios
       | updated_on  | NULL                                    |
       | document    | <document>                              |
       | created_on  | <timestamp>                             |
+
+  Scenario: Successfully create a Document Pointer for one directional data sync when date is provided
+    Given Producer "BaRS (EMIS)" (Organisation ID "V4T0L.YGMMC") is requesting to create Document Pointers
```

Comment:
Might be worth changing the names of the provider to something like Data Sync to avoid any confusion in the future seeing as this is a one off event